R/write_xlsx.R

```diff
@@ -1,20 +1,11 @@
-#' @name write_xlsx
-#' @title write data to an xlsx file
-#' @description write a data.frame or list of data.frames to an xlsx file
-#' @param x object or a list of objects that can be handled by [write_data()] to write to file
-#' @param file xlsx file name
-#' @param as_table write using write_datatable as opposed to write_data
-#' @param ... optional parameters to pass to functions:
-#' \itemize{
-#'   \item{[wb_workbook()]}
-#'   \item{[wb_add_worksheet()]}
-#'   \item{[wb_add_data()]}
-#'   \item{[wb_freeze_pane]}
-#'   \item{[wb_save()]}
-#' }
+# `write_xlsx()` ---------------------------------------------------------------
+#' Write data to an xlsx file
 #'
-#' see details.
-#' @details Optional parameters are:
+#' Write a data.frame or list of data.frames to an xlsx file
+#'
+#' @seealso [wb_add_worksheet()], [write_data()]
+#' @details
+#' Optional parameters are:
```

Comment:
Only changes in `write_xlsx()` is the order, and use of markdown for lists, see `write_xlsx.Rd` for the changes in the rendered docs